### PR TITLE
Updating step for BitBucket integration

### PR DESCRIPTION
**Pull request by hellotaran**
Default remote it 'origin' and not Magento. We could not add origin until we delete the existing.

## Purpose of this pull request

This pull request (PR) is to fix the steps for BitBucket integration.

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/integrations/bitbucket-integration.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/integrations/bitbucket-integration.md [hunk 1]
```diff
@@ -40,16 +40,22 @@ You must clone your {{site.data.var.ece}} project from an existing environment a
    magento-cloud project:get <project-ID>
    ```
 
-1. Add your Bitbucket repository as a remote.
+1. View remote repository URL.
```
Comment (nuzil) [inline src/cloud/integrations/bitbucket-integration.md:43]: I would re-phrase this into "Check existed remotes".

**Comment by nuzil** [comment]
@hellotaran please also check failed tests

File: src/cloud/integrations/bitbucket-integration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hguthrie) [reply src/cloud/integrations/bitbucket-integration.md:43]: This should fix the trailing space, too.

**Comment by meker12** [comment]
running tests

**Pull request merged**